### PR TITLE
fix(jsii-pacmak): handles 'default' doc component for go

### PR DESCRIPTION
**Pull request by sudolibre**
The default values were being parsed but not emitted for Go, effectively deleting them from the generated Go code. This was already handled for other languages. 

note: following the pattern seen for link, I added the extra call to `this.emitComment();`

resolves: [3885](https://github.com/aws/jsii/issues/3885)

---

By submitting this pull request, I confirm that my contribution is made under the terms of the [Apache 2.0 license].

[Apache 2.0 license]: https://www.apache.org/licenses/LICENSE-2.0

### CODE REVIEW

**Comment by jonathon-day** [comment]
Hopefully a quick fix here, @RomainMuller.

**Comment by jonathon-day** [comment]
@RomainMuller hopefully a quick fix here

**Comment by sudolibre** [comment]
@RomainMuller hopefully a quick fix here

**Comment by jonathon-day** [comment]
@RomainMuller had some test failures due mistakes with manually updating snapshot file. I discovered this is built into the tooling and updated the snapshot that way. Can you approve the workflows to run when you get a chance.

File: packages/jsii-pacmak/lib/targets/go/documentation.ts [hunk 1]
```diff
@@ -61,6 +61,11 @@ export class Documentation {
       this.emitComment();
     }
 
+    if (docs.default !== '') {
+      this.emitComment(`Default: ${docs.default}`);
+      this.emitComment();
```
Comment (RomainMuller) [inline packages/jsii-pacmak/lib/targets/go/documentation.ts:66]: Given the snapshot looks... I wonder if these two lines should be swapped?

**Review by RomainMuller — APPROVED** [review]

**Review by RomainMuller — APPROVED** [review]

**Comment by RomainMuller** [comment]
@mergifyio update

**Review by RomainMuller — APPROVED** [review]

**Pull request merged**